tokens/eth/0xDB5C3C46E28B53a39C255AA39A411dD64e5fed9c.json

```diff
@@ -0,0 +1,24 @@
+{
+    "symbol": "NCR",
+    "address": "0xdb5c3c46e28b53a39c255aa39a411dd64e5fed9c",
```

Comment:
this must be ERC55

Reply:
Thanks, i forgot to apply it here :) Done now